accepted/0000-fallback-kebab-case-command.md

```diff
@@ -0,0 +1,37 @@
+# Fallback to kebab-case command
+
+## Summary
+
+If the command is unknown, then fallback to the kebab-case version of it if it is in the `PATH`.
```

Comment:
so if i do `npm fooBar` you'd expect it to be the same as `npm foo-bar`?

Reply:
No, `npm fooBar` would result in searching for `npm-fooBar` and `npm foo-bar` would result in searching for `npm-foo-bar`.

Reply:
in that case, i wouldn't call it "the kebab-case version of it", i'd call it "the command prefixed with `npm-`".